src/views/index.njk

```diff
@@ -56,10 +56,25 @@
     <li><a href="/personal-tax/choose-paperless" class="govuk-link">Choose how to get your legal notices, penalty notices and tax letters</a></li>
   </ul>
 
-  <h3 class="govuk-heading-s">Clear app session</h2>
+  <h3 class="govuk-heading-s">Logout</h2>
 
   {{ govukButton({
       text: "Clear app session",
       href: "/login/clear-session"
   }) }}
+
+  <p class="govuk-body">To log out of the prototype you need to sign out of both Auth and the ESS Broker.</p>
+
+  {{ govukButton({
+      text: "Sign out from Auth (opens in new tab)",
+      href: "/login/logout-auth",
+      attributes: {"target": "_blank"}
+  }) }}
+
+  <p class="govuk-body"></p>
```

Comment:
@danacotoran there's got to be a better way of making these buttons appear on separate lines than putting an empty `<p>` tag between them, but my mind has gone completely blank. 

Any ideas?

Reply:
So I was thinking we'd wrap the whole thing in a `<div>` and then put a class of something like `display: block` on that.

But having just had a fiddle seems if you remove the govuk body tag they're separted 🤔 
![image](https://user-images.githubusercontent.com/3694062/185879021-074bc7f8-8615-4851-98ae-0fce709fa434.png)

Reply:
ah ha, wait no, I'm getting line wrap because the content is so long!

Reply:
Sorry, completely forgot about this until Huw started commenting on the thread.
If they are inline and you don't want them to me, I'd probably just wrap them in `<div>` as it doesn't have any particular semantic meaning and it is a block element.

Reply:
Great, thanks. I'll do that

Reply:
I made a patch here, not sure if flex wrap is overkill but it works:
```From f86be364a92286742f8dddc2e88146ca21a1b573 Mon Sep 17 00:00:00 2001
From: Huw Diprose <huw.diprose@digital.cabinet-office.gov.uk>
Date: Mon, 22 Aug 2022 09:54:44 +0100
Subject: [PATCH] Patch

---
 src/public/stylesheets/application.scss |  5 +++++
 src/views/index.njk                     | 24 ++++++++++++------------
 2 files changed, 17 insertions(+), 12 deletions(-)

diff --git a/src/public/stylesheets/application.scss b/src/public/stylesheets/application.scss
index 9efcd3e..a644009 100644
--- a/src/public/stylesheets/application.scss
+++ b/src/public/stylesheets/application.scss
@@ -154,3 +154,8 @@
   top: 2px;
 
 }
+
+.button-wrap {
+  display: flex;
+  flex-wrap: wrap;
+}
\ No newline at end of file
diff --git a/src/views/index.njk b/src/views/index.njk
index eca436d..420c510 100644
--- a/src/views/index.njk
+++ b/src/views/index.njk
@@ -65,16 +65,16 @@
 
   <p class="govuk-body">To log out of the prototype you need to sign out of both Auth and the ESS Broker.</p>
 
-  {{ govukButton({
-      text: "Sign out from Auth (opens in new tab)",
-      href: "/login/logout-auth",
-      attributes: {"target": "_blank"}
-  }) }}
-
-  <p class="govuk-body"></p>
-  {{ govukButton({
-      text: "Sign out from ESS broker (opens in new tab)",
-      href: "/login/logout-ess",
-      attributes: {"target": "_blank"}
-  }) }}
+  <div class="button-wrap">
+    {{ govukButton({
+          text: "Sign out from Auth (opens in new tab)",
+          href: "/login/logout-auth",
+          attributes: {"target": "_blank"}
+    }) }}
+    {{ govukButton({
+          text: "Sign out from ESS broker (opens in new tab)",
+          href: "/login/logout-ess",
+          attributes: {"target": "_blank"}
+    }) }}
+  </div>
 {% endblock %}
-- 
2.34.1

```

Reply:
Dana's suggestion of putting each button in its own `<div>` works too without any extra CSS or classes needed - see [caf8375](https://github.com/alphagov/di-solid-prototype/pull/142/commits/caf837549abc0bc464fdc7bcdc093d2ba3e3743b)

Reply:
Thanks Alex. I thought it was a bit weird that there isn't an option to display this design system element as block passing a flag or something, but I guess the reason they haven't done it is because it's simple enough to display any inline element as block by giving it a block wrapper.